modules/indigo/module/inc/indigo/of_state_manager.h

```diff
@@ -329,6 +407,24 @@ indigo_core_gentable_register(
 void
 indigo_core_gentable_unregister(indigo_core_gentable_t *gentable);
 
+/*
+ * @brief resume indigo addi4()/modifyi4()/del4() when the driver gets the final status 
```

Comment:
here's addi4 again...  let's do a search and fix all of those instances where it shows up.

Reply:
"resume a pending add4/modify/del4 when the asynchronous operation is complete"